Source/WebCore/workers/service/ServiceWorkerTypes.h

```diff
@@ -97,3 +97,17 @@ struct ServiceWorkerScripts {
 };
 
 } // namespace WebCore
+
+namespace WTF {
+
+template<> struct HashTraits<WebCore::ServiceWorkerOrClientIdentifier> : SimpleClassHashTraits<WebCore::ServiceWorkerOrClientIdentifier> {
```

Comment:
Is there a way to do this with less boilerplate that just makes a variant use the traits for its first variant option? Maybe that should be the default traits for any variant?